src/aux-server/aux-web/aux-auth/site/AuthApp/AuthApp.vue

```diff
@@ -152,7 +152,7 @@
             </md-app-content>
         </md-app>
 
-        <md-dialog :md-active.sync="showCreateStudio">
+        <md-dialog :md-active.sync="showCreateStudio" @keypress.enter="createStudio()">
```

Comment:
@TroyceGowdy This dialog has a form that should catch when people use the enter key to submit the form. Is this change necessary or am I incorrect?